src/platform_impl/windows/event_loop.rs

```diff
@@ -1587,6 +1604,34 @@ unsafe extern "system" fn public_window_callback<T>(
                         continue;
                     }
 
+                    let force = match pointer_info.pointerType {
+                        winuser::PT_TOUCH => {
+                            let mut touch_info: winuser::POINTER_TOUCH_INFO = mem::uninitialized();
```

Comment:
We've moved away from using `mem::uninitialized`, since it's unclear if it's possible to use safely. Could you replace this with `MaybeUninit`?

Reply:
Sure. I've seen the efforts to replace `mem::unitialized` but didn't know if it's finalized or not.